src/js/components/NasaTable.js

```diff
@@ -0,0 +1,117 @@
+    }
+
+    prevPage = async () => {
+        const { rowsPerPage, page, data } = this.state
+        const from = rowsPerPage * (page - 1)
+        const visibleRows = this.getDataToShow(data.table, from, rowsPerPage)
+        this.setState({ visibleRows, page: page - 1 })
+    }
+
+    changeRow = async (e) => {
+        const { data } = this.state
+        const rows = +e.target.value
+        const visibleRows = this.getDataToShow(data.table, 0, rows)
+        this.setState({ rowsPerPage: rows, visibleRows, page: 0 })
+    }
+
+    fillCells = (cell, i) => {
+        if (i !== 0) {
+            return (<TableCell key={i}>{ cell }</TableCell>)
+        }
+        return (
+            <TableCell
+                key={i}
+                onClick={this.openDialog}
+                onMouseEnter={ () => this.setState({onLink: true})}
+                onMouseLeave={ () => this.setState({onLink: false})}
+                className={`first-item ${this.state.onLink && 'on-link'}`}
+            >
+                { cell }
+            </TableCell>)
+    }
+
+    openDialog = () => this.setState({ open: true })
+
+    componentDidMount = async () => {
+        const { rowsPerPage, page } = this.state;
+        const res = await getNasaData();
+        const data = await res.json();
+        const from = rowsPerPage * page
+        const visibleRows = this.getDataToShow(data.table, from, rowsPerPage)
+        this.setState({ data, visibleRows })
+    }
+
+    render() {
+        const { data, page, rowsPerPage, visibleRows, open } = this.state
+        const { headers, table = [] } = data
+        return(
+            <React.Fragment>
+                <ImageDialog open={ open } onClose={ this.handleClose }/>
+                <Table>
+                    <TableHead>
+                        <TableRow>
+                            { headers && headers.map((el, i) => <TableCell  className='table-headers' key={i}>{ el }</TableCell>)}
+                        </TableRow>
+                    </TableHead>
+                    <TableBody>
+                        { visibleRows && visibleRows.map((row, i) => <TableRow key={i}>
+                            { row.info.map((cell, i) => this.fillCells(cell, i)) }
+                        </TableRow>) }
+                    </TableBody>
+                </Table>
+                <TablePagination
+                    rowsPerPageOptions={[5, 10, 25]}
+                    component='div'
+                    count={table && table.length}
+                    rowsPerPage={rowsPerPage}
+                    page={page}
+                    onChangeRowsPerPage={this.changeRow}
+                    backIconButtonProps={{
+                        'aria-label': 'Previous Page',
+                        'onClick': this.prevPage
+                    }}
+                    nextIconButtonProps={{
+                        'aria-label': 'Next Page',
+                        'onClick': this.nextPage
```

Comment:
In JavaScript we have keys of string type by default. You don't need to wrap them with quotes

Reply:
Okay, I'll fix it